Berksfile

```diff
@@ -3,5 +3,5 @@ source 'https://supermarket.chef.io'
 metadata
 
 group :integration do
-  cookbook 'test', path: 'test/fixtures/cookbooks/test'
+  cookbook 'test_java', path: 'test/fixtures/cookbooks/test'
```

Comment:
any particular reason for changing this?

Reply:
because lots of places reference it, I decided to be consistent with the `.kitchen.yml`